package.json

```diff
@@ -18,7 +18,9 @@
         "theme": "light"
```

Comment:
Please bump package/package-lock version

Reply:
Note that the release process should be separate and is documented in its own file: https://github.com/microsoft/azure-pipelines-vscode/blob/main/RELEASE.md

Reply:
@50Wliu got it, thanks!

Reply:
Hi @50Wliu,
It looks like we don't have enough rights to create a release. Could you help us with this?